### PR TITLE
Add Linode as an installation option in docs

### PR DESCRIPTION
**Pull request by techknowlogick**
as title

### CODE REVIEW

**Review by lunny — APPROVED** [review]

File: docs/content/doc/installation/on-cloud-provider.md [hunk 1]
```diff
@@ -35,10 +35,16 @@ you can experiment with running Gitea.
 
 Gitea can found in [Vultr](https://www.vultr.com)'s marketplace.
 
-To deploy it have a look at https://www.vultr.com/marketplace/apps/gitea.
+To deploy Gitea to Vultr, have a look at [the Vultr Marketplace listing](https://www.vultr.com/marketplace/apps/gitea).
```
Comment (delvh) [inline docs/content/doc/installation/on-cloud-provider.md]: Personally, I would put the `the` outside of the link in all three cases…

**Review by zeripath — APPROVED** [review]

File: docs/content/doc/installation/on-cloud-provider.md (hunk 1 above)
Comment (zeripath) [reply docs/content/doc/installation/on-cloud-provider.md]:
```suggestion
To deploy Gitea to Vultr, have a look at the [Vultr Marketplace](https://www.vultr.com/marketplace/apps/gitea).
```

File: docs/content/doc/installation/on-cloud-provider.md [hunk 2]
```diff
@@ -35,10 +35,16 @@ you can experiment with running Gitea.
...
 
 ## DigitalOcean
 
-[DigitalOcean](https://www.digitalocean.com) has gitea as droplet in his marketplace.
+[DigitalOcean](https://www.digitalocean.com) has Gitea as droplet in their marketplace.
 
-Just create a new [Gitea Droplet](https://marketplace.digitalocean.com/apps/gitea).
+To deploy Gitea to DigitalOcean, have a look at [the DigitalOcean Marketplace listing](https://marketplace.digitalocean.com/apps/gitea).
```
Comment (zeripath) [inline docs/content/doc/installation/on-cloud-provider.md]:
```suggestion
To deploy Gitea to DigitalOcean, have a look at the [DigitalOcean Marketplace](https://marketplace.digitalocean.com/apps/gitea).
```

File: docs/content/doc/installation/on-cloud-provider.md [hunk 3]
```diff
@@ -35,10 +35,16 @@ you can experiment with running Gitea.
...
+
+## Linode
+
+[Linode](https://www.linode.com/) has Gitea as an app in their marketplace.
+
+To deploy Gitea to Linode, have a look at [the Linode Marketplace listing](https://www.linode.com/marketplace/apps/linode/gitea/).
```
Comment (zeripath) [inline docs/content/doc/installation/on-cloud-provider.md]:
```suggestion
To deploy Gitea to Linode, have a look at the [Linode Marketplace](https://www.linode.com/marketplace/apps/linode/gitea/).
```

**Review by delvh — APPROVED** [review]

**Pull request merged**